ee/vellum_ee/workflows/display/nodes/vellum/inline_prompt_node.py

```diff
@@ -228,9 +249,15 @@ def _generate_prompt_block(
 
     def _serialize_attributes(self, display_context: "WorkflowDisplayContext"):
         attributes = []
+
+        blocks_contain_descriptors = getattr(self, "_blocks_contain_descriptors", False)
+
         for attribute in self._node:
             if attribute in self.__unserializable_attributes__:
-                continue
+                if attribute == InlinePromptNode.blocks and blocks_contain_descriptors:
```

Comment:
Instead of making an exception here, move `InlinePromptNode.blocks` from `__unserializable_attributes__` to `__serializable_attributes__`